File: Programas em Phyton/Ex36.py
numeros = [20]
constante = 5
for i in range (0, 20, 1):
    x = int(input('Digite 20 números: '))
    numeros.append(x)
for i in range (0, 20, 1):
    for j in range (i+1, 20, 1):
        if(numeros[i]>numeros[j]):
            aux = numeros[i]
            numeros[i]=numeros[j]
            numeros[j]=aux
print("Os números multiplicados e ordenados são: \n")
for i in range (-1, 19, 1):
    numeros[i] = constante * numeros[i]
    print(f'{numeros[i]}')        

File: Programas em C/Ex36.c
#include<stdio.h>
#include<stdlib.h>
#include<ctype.h>
int main(int argc, char const *argv[])
{
    int numeros[20], j, aux;
    int constante;
    constante=5;

    printf("Digite os 20 numeros: ");
    for(int i=0; i<20; i++)
    {
        scanf("%i", &numeros[i]);
    }
    for(int i=0; i<20; i++)
    {
        for(int j=i+1; j<20; j++)
        {
            if(numeros[i]>numeros[j])
            {
                aux = numeros[i];
                numeros[i]=numeros[j];
                numeros[j]=aux;
            }
        }
    }
    printf("Os numeros multiplicados e ordenados: \n");
    for(int i=0; i<20; i++)
    {
        if(numeros[i] = constante * numeros[i]);
        printf("%i\n", numeros[i]);
    }
    return 0;
}
